fresh_mm_bot/src/helpers/match.ts

```
@@ -0,0 +1,83 @@
+        const winScore = await getWinScore();
+        if (!match.teamARounds || !match.teamBRounds) return;
+
+        const mostRounds = match.teamARounds > match.teamBRounds ? 'a' : 'b';
+        const winner =
+            match.gameType === GameType.squads
+                ? match.teamARounds === winScore
+                    ? 'a'
+                    : 'b'
+                : mostRounds;
+        resolve(winner);
+    });
```

Biome (2.4.10)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)